erp-client/src/app/app.component.html

```diff
@@ -1,6 +1,6 @@
 <mat-toolbar color="primary">
   <mat-toolbar-row>
-    <img [src]="'assets/Images/prototype-logo.png'" class="login-logo">
+    <img [src]="'assets/Images/prototype-logo.svg'" class="login-logo">
```

Comment:
I recreated the logo in Sketch as an SVG so it could have a transparent background and be vectorized. Not sure if we were planning to stick w/ this logo or not (no preference here). Feel free to revet to the PNG if that's preferred.

Reply:
No attachments to our existing logo. Looks good.